src/cryptoadvance/specterext/stacktrack/controller.py

```diff
@@ -44,19 +47,67 @@ def index():
     return render_template("stacktrack/index.jinja")
 
 
-# @stacktrack_endpoint.route("/wallet_chart", methods=["GET"])
+@stacktrack_endpoint.route("/wallets_overview")
+@login_required
+def wallets_overview():
+    span: str = request.args.get("span")
+    span = "1y" if span is None else span
+
+    specter().check_blockheight()
+
+    wallets_overview_vm_dict = specter().service_manager.execute_ext_callbacks(
+        adjust_view_model, WalletsOverviewVm()
+    )
+    if len(wallets_overview_vm_dict) == 0:
+        wallets_overview_vm = WalletsOverviewVm()
+    elif len(wallets_overview_vm_dict.values()) == 1:
+        wallets_overview_vm = list(wallets_overview_vm_dict.values())[0]
+    else:
+        raise logger.error("Seems that we have more than one WalletsOverviewVm Extension")
```

Comment:
Oh, i see, you copied the templates from "core" and now you need a `wallet_overview_vm` and the way you do it is simply calling this here. I think this should also work without calling the `execute_ext_callbacks` ?! Or maybe make your desired change here directly and not use that callback?!

Reply:
Oh, got it. Yeah, invoking the callback here is an unnecessary complication. Will fix.